src/dataValidation/checkValidData.js

```diff
@@ -0,0 +1,37 @@
+const availableCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+const errosMassega = {
+  emptyText() {
```

Comment:
why'd you need a function here?

Reply:
> error handling should pass the tests if it does not now

@mykola-koval  
![image](https://user-images.githubusercontent.com/23125219/209842831-999a2cc8-2217-482f-8775-b6f7fe8a945e.png)

Reply:
> why'd you need a function here?

i didn't understand a question, i want to have one collection with functions to call it simplify